learning/tour-of-beam/frontend/lib/pages/tour/widgets/group_title.dart

```diff
@@ -74,19 +75,7 @@ class _GroupProgressIndicator extends StatelessWidget {
           );
         }
 
-        return Container(
-          margin: const EdgeInsets.symmetric(horizontal: BeamSizes.size6),
-          height: BeamSizes.size8,
-          width: BeamSizes.size8,
-          child: CircularProgressIndicator(
-            strokeWidth: BeamSizes.size3,
-            color: BeamColors.green,
-            backgroundColor: Theme.of(context)
-                .extension<BeamThemeExtension>()!
-                .unselectedProgressColor,
-            value: progress,
-          ),
-        );
+        return TobProgressIndicator(progress: progress);
```

Comment:
Maybe name them `BooleanProgressIndicator` and `DoubleProgressIndicator`? I haven't looked at this code for 2 weeks and have already forgotten what `TobProgressIndicator` does.

Reply:
"Double" confused me, so I renamed it to `ArbitraryProgressIndicator`. But I don't like the word "arbitrary" either.

Reply:
`FractionProgressIndicator`.